hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/DefinitionsTest.java

```diff
@@ -72,51 +78,538 @@ public void test000InitScheme() throws Exception {
     }
 
     @Test
-    public void test001CopyTrigger() throws Exception {
-        Trigger t = definitionsService.getTrigger(TEST_TENANT, "trigger-1");
-        assertTrue(t != null);
+    public void test0010GroupTrigger() throws Exception {
```

Comment:
Number on tests were provided when we can't delete alerts so easily, so perhaps as an improvement for future we can validate if tests are isolated and we can stop using number for ordering.
It's just a note to remind it in future.